src/main/scala/chiseltest/TestResult.scala

```diff
@@ -0,0 +1,9 @@
+// SPDX-License-Identifier: Apache-2.0
+
+package chiseltest
+
+import firrtl.AnnotationSeq
+
+class TestResult(annotations: AnnotationSeq) {
```

Comment:
@ducky64 Is there a good file to put the `TestResult` class in? (Instead of creating a new file)

Reply:
I think having its own file is fine? It is (or I guess could be) a significant API, even though the implementation is super short?

Reply:
I am happy with that.
Any good ideas for a non-obvious scala doc?
I was trying, but all I could come up with was:
> Contains the results of a test run.

Reply:
Perhaps also mention that this is meant for future expansion of the test return API, though there's not much in it right now.

Reply:
(I could totally see someone asking "wait why does this even exist")

Reply:
I will add a plain comment, since this is mostly developer and not user facing.

Reply:
added the comment